docs/Información del servicio/Story board.mdx

```diff
@@ -0,0 +1,124 @@
+## Historial de versiones
+
+| Versión | Fecha      | Diferencia entre versiones                                                                                     |
+| ------- | ---------- | ---------------------------------------------------------------------------------------------------------------|
+| 1.0     | 2024-03-14 | Versión inicial del documento.                                                                                 |
+| 2.0     |            |                                                    |
```

Comment:
In the document there is an empty row in the version history, this row must be deleted.

Reply:
I've corrected the mistakes, I'm waiting for the feedback @alvarobernal2412 .